app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, original_file, file_to_copy = command.split()
+
+    if not (len(command.split()) < 3
+            or original_file == file_to_copy
+            or command != "cp"):
+
+        with open(original_file, "r") as orig, open(file_to_copy, "a") as copy:
```

Comment:
I'd suggest using the whole word if you aren't completely sure it's necessary. Use brackets if you need to split in lines